docs/source/autograd.rst

```diff
@@ -14,6 +14,26 @@ Automatic differentiation package - torch.autograd
     backward
     grad
 
+.. forward-mode-ad:
+
+Forward-mode Automatic Differentiation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    This API is in beta. Even though the function signatures are very unlikely to change, improved
+    operator coverage is planned before we consider this stable.
+
+Please see our `forward-mode AD tutorial <https://pytorch.org/tutorials/intermediate/forward_ad_usage.html>`__
+for detailed steps on how to use this API.
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    forward_ad.dual_level
+    forward_ad.make_dual
```

Comment:
I think the short descriptions are almost perfect. I would still define what "tangent" means in there by having a (the gradient) added for this and the one below.